src/run.ts

```diff
@@ -29,14 +29,23 @@ abstract class RenderEngine {
 class HelmRenderEngine extends RenderEngine {
     public bake = async (): Promise<any> => {
         const helmPath = await getHelmPath();
-        core.debug("Creating the template argument string..");
-        var args = this.getTemplateArgs()
-         const options = {
+
+        const chartPath = core.getInput('helmChart', {required : true});
+        const options = {
             silent: true
-         } as ExecOptions;
+        } as ExecOptions;
+
+        var dependencyArgs = this.getDependencyArgs(chartPath);
+        
+        core.debug("Running helm dependency update command..");
+        await utilities.execCommand(helmPath, dependencyArgs, options);
+
+        core.debug("Creating the template argument string..");
+        var args = this.getTemplateArgs(chartPath)
         
         core.debug("Running helm template command..");
```

Comment:
This too.

Reply:
This was core.debug already. Should I still update it?